server/classes/sockets/CommonSocketBinder.ts

```diff
@@ -1,29 +1,43 @@
 import { Socket } from 'socket.io';
+import ISession from '../../interfaces/ISession';
```

Comment:
Je met le commentaire ici, même si il ne concerne pas que cette partie. Il y'a un gros problème de cohérence au niveau des quotes pour les strings : Parfois des single, parfois des doubles... Il faut se décider 😛 . Le linter est pas censé vous restreindre la dessus ?

Reply:
Linter isn't that sharp, I'm opening an issue to add the rule and enforce it.